scripts/app

```diff
@@ -106,6 +116,11 @@ update_installed_apps() {
   echo "${updated_json}" > "${USER_FILE}"
 }
 
+# Make sure no other install script is running, if it is, wait for it to exit
+while [ $(check_if_not_already_running) == 2]; do 
```

Comment:
```suggestion
while [[ "$(check_if_not_already_running)" == "2" ]]; do 
```

Some suggestions:
- Prefer `[[` over `[` to prevent weird behaviour
- Always quote strings in bash to prevent weird behaviour